python/dgl/distributed/dist_graph.py

```diff
@@ -369,7 +369,7 @@ def _init(self):
         self._client.map_shared_data(self._gpb)
 
     def __getstate__(self):
-        return self.ip_config, self.graph_name, self._gpb_input
+        return self.ip_config, self.graph_name, self._gpb
```

Comment:
Is any change needed in the `__setstate__` function or `_init` function for this?

Reply:
it doesn't. previously, `_gpb_input` may just be None. `_gpb` contains the real graph partition book.